src/game-asset.lisp

```diff
@@ -49,6 +49,13 @@
     (when load-now (load-asset asset))
     asset))
 
+(defmethod copy-asset-to-object ((asset image-asset))
+  (load-asset asset)
+  (let* ((image (make-instance 'rl-image))
+         (ptr (autowrap:ptr (c-struct image))))
+    (claylib/ll:image-copy ptr (c-asset asset))
```

Comment:
Any reason you used the pointer here instead of just copying into the c-struct? Not saying it's wrong. I just don't _think_ it should be necessary and was wondering if you had information to counter that assumption.